app/views/layouts/body/_title.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="container site-title-container">
+  <%= link_to home_path do %>
+    <div class="h1">
+      <% if I18n.exists?("general.project_name", locale) %>
+        <span class="site-title"><%= t "general.project_name" %></span>
+      <% end %>
+      
+      <!-- no subtitle -->
```

Comment:
delete?

Reply:
I would like to leave in until I check with Lorna, but I changed to rails style comments.

Reply:
never mind, I can just delete the template